Lib/xml/dom/minidom.py

```diff
@@ -1110,7 +1117,7 @@ def splitText(self, offset):
         return newText
 
     def writexml(self, writer, indent="", addindent="", newl=""):
-        _write_data(writer, "%s%s%s" % (indent, self.data, newl))
+        _write_text_data(writer, "%s%s%s" % (indent, self.data, newl))
```

Comment:
I haven't checked deeply, but my gut feeling would prefer a special case for attribute values, not for text content. Can't say if that's similarly easy to achieve.